ServiceRepositoryPattern/ServiceRepositoryPattern/Controllers/FoodItemsController.cs

```diff
@@ -0,0 +1,152 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.EntityFrameworkCore;
+using ServiceRepositoryPattern.Data;
+using ServiceRepositoryPattern.Models;
+using ServiceRepositoryPattern.Services;
+
+namespace ServiceRepositoryPattern.Controllers
+{
+    public class FoodItemsController : Controller
+    {
+        private readonly ServiceRepositoryPatternContext _context;
+
+        public FoodItemsController(ServiceRepositoryPatternContext context)
+        {
+            _context = context;
+        }
+
+        public async Task<IActionResult> Index()

```

Comment:
WHy you need index method in each controller? what it does? If its a common method maybe create a base class for that or put it into utility based on its behavior.

Reply:
For the first page of the controller.